README.md

```diff
@@ -12,20 +12,21 @@ than [llama-server](https://github.com/ggerganov/llama.cpp/blob/master/examples/
+| NVIDIA CUDA 12.5-l         | `linux/amd64` `windows/amd64` | Compute capability is `6.0`, `6.1`, `7.0`, `7.5` ,`8.0`, `8.6` or `8.9`, see https://developer.nvidia.com/cuda-gpus.                                                                                                    |
+| AMD ROCm/HIP 5.5-s         | `windows/amd64`               | LLVM target is `gfx1030`, `gfx1100`, `gfx1101` or `gfx1102`, see https://rocm.docs.amd.com/en/docs-5.5.1/release/windows_support.html.                                                                                  |
+| AMD ROCm/HIP 5.7-s         | `linux/amd64` `windows/amd64` | LLVM target is `gfx1030`, `gfx1100`, `gfx1101` or `gfx1102`, see https://rocm.docs.amd.com/en/docs-5.7.1/release/gpu_os_support.html, https://rocm.docs.amd.com/en/docs-5.7.1/release/windows_support.html.             |
+| AMD ROCm/HIP 5.7-l         | `windows/amd64`               | LLVM target is `gfx900`, `gfx906`,`gfx908`, `gfx90a`, `gfx940`, `gfx1030`, `gfx1100`, `gfx1101` or `gfx1102`, see https://rocm.docs.amd.com/en/docs-5.7.1/release/windows_support.html.                                 |
+| AMD ROCm/HIP 6.1-s         | `linux/amd64`                 | LLVM target is `gfx1030`, `gfx1100`, `gfx1101` or `gfx1102`, see https://rocm.docs.amd.com/en/docs-5.5.1/release/windows_support.html.                                                                                  |
+| AMD ROCm/HIP 6.1-l         | `linux/amd64`                 | LLVM target is `gfx900`, `gfx906`,`gfx908`, `gfx90a`, `gfx940`, `gfx1030`, `gfx1100`, `gfx1101` or `gfx1102`, see https://rocm.docs.amd.com/projects/install-on-linux/en/docs-6.1.2/reference/system-requirements.html. |
+| Intel oneAPI 2024.1        | `linux/amd64` `windows/amd64` | See Linux/Windows distributions with oneAPI 2024.1 from https://www.intel.com/content/www/us/en/developer/articles/system-requirements/intel-oneapi-base-toolkit-system-requirements.html.                              |
+| Intel oneAPI 2024.2        | `linux/amd64` `windows/amd64` | See Linux/Windows distributions with oneAPI 2024.2 from https://www.intel.com/content/www/us/en/developer/articles/system-requirements/intel-oneapi-base-toolkit-system-requirements.html.                              |
+| Ascend CANN 8.0            | `linux/amd64` `linux/arm64`   | See https://www.hiascend.com/document/detail/zh/CANNCommunityEdition/80RC3alpha001/quickstart/quickstart/quickstart_18_0002.html.                                                                                       |
+| Moore Threads MUSA 1.5.1   | `linux/amd64`                 | `MTT S4000`.                                                                                                                                                                                                            |
```

Comment:
can we provide a document or link here? so users can get more information from here.

Reply:
This document is still a work in progress. I can update it later in a new PR, or we can remove it for now. Which option do you prefer?

Reply:
I think you can update it later. thanks.